statistics/cmsketch.go

```diff
@@ -362,6 +391,26 @@ func (c *CMSketch) Equal(rc *CMSketch) bool {
 			}
 		}
 	}
+	if len(c.topN) != len(rc.topN) {
+		return false
+	}
+	for k, topNData := range c.topN {
+		if len(topNData) != len(rc.topN[k]) {
+			return false
+		}
+		for _, val := range topNData {
+			found := false
+			for _, val2 := range rc.topN[k] {
+				if val.h2 == val2.h2 && bytes.Equal(val.data, val2.data) {
```

Comment:
Why we don't check count and h1?

Reply:
Seems we can even remove h1 from topNMeta